modules/candidate_parameters/php/candidatequeryengine.class.inc

```diff
@@ -207,6 +265,16 @@ class CandidateQueryEngine extends \LORIS\Data\Query\SQLQueryEngine
                 . ' ON (c.RegistrationCenterID=rsite.CenterID)'
             );
             return 'rsite.Name';
+        case 'DateRegistered':
+            $this->addTable(
```

Comment:
Is there a reason these fields need to join session multiple times instead of just copy/pasting the join from line 258?

You're just getting different fields from the same table